airflow/lineage/backend/atlas/__init__.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
```

Comment:
For me it makes more sense that we have an interface that is extended by an implementation of a lineage backend.

Reply:
You mean using an abstract class? It looks like overkill right now (only one function is called), but can do